{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/sections/models.py

```diff
@@ -39,17 +40,13 @@
     }),
     ('Text', {
         'sections': [
-            ('centered', {
-                'fields': ['background_colour', 'kicker', 'title', 'text', 'link_text', 'link_page', 'link_url'],
+            ('wysiwyg', {
+                'name': 'Rich text',
+                'fields': ['background_colour', 'kicker', 'title', 'content', 'link_text', 'link_page', 'link_url'],
                 'search': ['kicker', 'title', 'text'],
```

Comment:
You've changed from text to content but not updated the search fields

Reply:
Good catch, thank you.